### PR TITLE
Deduplicate alerting settings in proto

### PR DESCRIPTION
**Pull request by chases2**
Addresses #36

Deduplicates alerting options to be contained in a Dashboard Tab instead of the underlying Test Group.
This gives an authoritative place where these setting can be stored or read from, and makes explicit an intent to migrate in this direction.

### CODE REVIEW

File: pb/config/config.proto [hunk 1]
```diff
@@ -607,6 +604,14 @@ message DashboardTabAlertOptions {
 
   // Custom text to show for the debug link.
   string debug_message = 7;
+
+  // Wait time between emails. If unset or zero, an email will be sent only once
+  // it becomes a consistent failure, and not again until it succeeds.
+  // TestGrid does not pester about staleness
```
Comment (michelle192837) [inline pb/config/config.proto:610]: nit: Maybe clarify "Regardless of how this is set, TestGrid does not pester about staleness" ?

Comment (michelle192837) [reply pb/config/config.proto:610]: Also is this true?

**Review by michelle192837 — APPROVED** [review]
/hold

Unhold at will!

**Comment by michelle192837** [comment]
Actually, updating that to:

/hold

Unhold once internal config.proto is updated.

File: pb/config/config.proto (hunk 1 above)
Comment (chases2) [reply pb/config/config.proto:610]: I think this is true in the current implementation. Is it supposed to be different?

Comment (michelle192837) [reply pb/config/config.proto:610]: Nope, think that's fine, just wasn't sure. Carry on!

**Pull request merged**